consensus/polybft/polybft_config.go

```diff
@@ -262,3 +264,10 @@ func (m *Manifest) Save(manifestPath string) error {
 
 	return nil
 }
+
+// TokenConfig is the configuration of nativ token used by edge network
+type TokenConfig struct {
+	TokenName     string `json:"tokenName"`
+	TokenSymbol   string `json:"tokenSymbol"`
+	TokenDecimals uint8  `json:"tokenDecimals"`
```

Comment:
Maybe we can shorten these fields, as it is part of `TokenConfig`?
```suggestion
	Name     string `json:"name"`
	Symbol   string `json:"symbol"`
	Decimals uint8  `json:"decimals"`
```

Reply:
Fixed: https://github.com/0xPolygon/polygon-edge/pull/1376/commits/ab7a094805b4e5da93a395b0dd84f32cf7f251e8